Source/App/EbAppInputy4m.c

```diff
@@ -46,10 +46,10 @@ int32_t read_y4m_header(EbConfig_t *cfg) {
 	assert(fresult != NULL);
 
 	/* print header */
-	if (PRINT_HEADER) {
-		printf("y4m header:");
-		fputs(buffer, stdout);
-	}
+#ifdef PRINT_HEADER
```

Comment:
indent is still messed up. Pls fix them. Thx.

Reply:
done..